pkg/defaults/policies/files/drop_all_capabilities.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "dd478ad6-b9c9-4abf-92e0-847408aecb8d",
+  "name": "Drop All Capabilities",
+  "description": "Alert when a deployment does not drop all capabilities.",
+  "rationale": "Since capabilities permit privileged operations, it is a recommended best practice to drop all capabilities that a deployment can have, and then add only the capabilities that the deployment needs.",
+  "remediation": "Ensure that the deployment manifest has drop: ALL in the securityContext section of the Container manifest.",
```

Comment:
- Do you want to set off "drop: ALL" by tick marks or some other way for readability?
- Container doesn't need caps